src/addagent/manage_agents.c

```diff
@@ -220,9 +240,80 @@ int add_agent(int json_output)
             _ip = NULL;
             c_ip.ip = NULL;
         } else if (!authd_running && (id_exist = IPExist(ip))) {
-            double antiquity = OS_AgentAntiquity_ID(id_exist);
+            bool replace_agent = false;
+            char error_message[OS_SIZE_128];
+
+            cJSON *j_agent_info = NULL;
+            j_agent_info = wdb_get_agent_info(atoi(id_exist), NULL);
+
+            if (!j_agent_info){
+                merror_exit("Failed to get agent info for agent '%s'", id_exist);
```

Comment:
`Failed to get agent-info for agent` instead of `Failed to get agent info for agent`